priv/repo/seeds.exs

```diff
@@ -143,3 +161,29 @@ units =
   end)
 
 Repo.insert_all(Unit, units, on_conflict: :nothing)
+
+currency_rewards =
+  Enum.map(Enum.with_index(levels_without_units, 0), fn {level, level_index} ->
+      %{
+        level_id: level.id,
+        amount: 10 * level_index,
+        currency_id: gold_currency.id,
+        inserted_at: NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second),
+        updated_at: NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second)
+      }
+
+  end)
+  Repo.insert_all(CurrencyReward, currency_rewards, on_conflict: :nothing)
+
+afk_reward_incrementers =
+  Enum.map(Enum.with_index(levels_without_units, 0), fn {level, level_index} ->
```

Comment:
This should start on 1 because the first level should already give some kind of reward!